dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/FileImporter.java

```diff
@@ -181,8 +181,9 @@ private void handleMap(Entry entry, SourceFile.FileType fileType, Version versio
             Map<String, ?> map) {
         for (Map.Entry<String, ?> e : map.entrySet()) {
             final Object mapValue = e.getValue();
-            if (e.getKey().equalsIgnoreCase("$import") || e.getKey().equalsIgnoreCase("$include") || e.getKey().equalsIgnoreCase("import")
-                    || e.getKey().equalsIgnoreCase("include")) {
+            if (e.getKey().equalsIgnoreCase("$import") || e.getKey().equalsIgnoreCase("$include") || e.getKey().equalsIgnoreCase("$mixin")
+                || e.getKey().equalsIgnoreCase("import") || e.getKey().equalsIgnoreCase("include") || e.getKey()
+                .equalsIgnoreCase("mixin")) {
```

Comment:
Making an array of those terms and then array.contains(e.getKey().toLowercase()) might look prettier.